src/utils/validation.ts

```diff
@@ -0,0 +1,24 @@
+import { ValidationRules } from '../../typings/commonTypes';
+import { emailRegExp, phoneRegExp } from './regex';
```

Comment:
🧐
Тут тоже неплохо бы задуматься об алиасе для директории с типами
```suggestion
import { ValidationRules } from 'typings/commonTypes';
import { emailRegExp, phoneRegExp } from './regex';
```

Reply:
ValidationRules type перенёс в компонент Input. В commonTypes остались только RoutesList, но пока что у меня все роуты в App, я типы не переношу. Позже наверное вынесу роуты в отдельный компонент и рядом положу типы

На счёт алиасов решили с Димой пока не использовать